static/js/code-highlighter.js

```diff
@@ -59,7 +59,7 @@ $("#scrolling").on('scroll', function() {
       return null;
     }
 
-    let num = parseInt(elem.dataset.lineNumber, 10);
+    let num = parseInt(elem.id.slice(1), 10);
```

Comment:
Wood it be better to use `elem.textContent` here? In case we end up changing the id later to just be the line number. (Ditto next hunk)

Reply:
Hm, yeah, that's a good idea.  I initially avoided doing that because the `scrollIntoView` hack creates a div as a child of this element and although it doesn't perturb textContent (it's an empty div), it would be bad if it did.  But in point-of-fact, we would want textContent to change and break because it's the textContent that gets surfaced to screen readers.

Reply:
Note that there are a ton of locations in the code right now that use the id, but that happens via the `lineFromId` helper in the first ready block at the bottom of the file.  So those will need to be changed, but the helper makes it obvious what needs to change.